### PR TITLE
Fix clippy::must_use_candidate (auto)

### PR DESCRIPTION
**Pull request by uklotzde**
Automatic fix.

### CODE REVIEW

File: src/musepack/sv7/properties.rs [hunk 1]
```diff
@@ -47,7 +47,7 @@ impl Profile {
 	///
 	/// The mapping is available here: <http://trac.musepack.net/musepack/wiki/SV7Specification>
 	#[rustfmt::skip]
-	pub fn from_u8(value: u8) -> Option<Self> {
+	#[must_use]	pub fn from_u8(value: u8) -> Option<Self> {
```
Comment (Serial-ATA) [inline src/musepack/sv7/properties.rs:50]: Formatting issue

File: src/lib.rs [hunk 2]
```diff
@@ -118,7 +118,6 @@
 	clippy::cast_possible_wrap,
 	clippy::cast_possible_truncation,
 	clippy::module_name_repetitions,
-	clippy::must_use_candidate,
```
Comment (Serial-ATA) [inline src/lib.rs:121]: I'd rather keep this `allow`ed, it's a pretty noisy lint when trying to iterate.

File: src/tag/item.rs [hunk 3]
```diff
@@ -436,7 +436,7 @@ macro_rules! gen_item_keys {
 			///
 			/// NOTE: If used with ID3v2, this will only check against the ID3v2.4 keys.
 			/// If you wish to use a V2 or V3 key, see [`upgrade_v2`](crate::id3::v2::upgrade_v2) and [`upgrade_v3`](crate::id3::v2::upgrade_v3)
-			pub fn from_key(tag_type: TagType, key: &str) -> Self {
+			#[must_use] pub fn from_key(tag_type: TagType, key: &str) -> Self {
```
Comment (Serial-ATA) [inline src/tag/item.rs:439]: Formatting

File: src/tag/item.rs [hunk 4]
```diff
@@ -449,7 +449,7 @@ macro_rules! gen_item_keys {
 			///
 			/// Use `allow_unknown` to include [`ItemKey::Unknown`]. It is up to the caller
 			/// to determine if the unknown key actually fits the format's specifications.
-			pub fn map_key(&self, tag_type: TagType, allow_unknown: bool) -> Option<&str> {
+			#[must_use] pub fn map_key(&self, tag_type: TagType, allow_unknown: bool) -> Option<&str> {
```
Comment (Serial-ATA) [inline src/tag/item.rs:452]: Formatting

**Review by Serial-ATA — CHANGES_REQUESTED** [review]

**Comment by uklotzde** [comment]
Probably not worth the noose when keeping the allow-exception.

**Comment by Serial-ATA** [comment]
Yeah, I'll remove the `allow` at a later date, when stuff is more finalized. Right now Lofty moves too fast with new additions that the lint becomes kind of an annoyance.

**Pull request closed without merging**